source/faq/faq.rst

```diff
@@ -259,95 +303,94 @@ Data Flows
 What is data flow?
 ^^^^^^^^^^^^^^^^^^
 
-It’s a processing flow for income messages/data of a device. With a flow you can dynamically 
-analyse each new message in order to apply validations, infer information, and trigger 
-actions or notifications. 
+It’s a processing flow for income messages/data of a device. With a flow you
+can dynamically analyse each new message in order to apply validations, infer
+information, and trigger actions or notifications.
 
 The data flow UI… really looks like node-RED. Are they related in some way?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It’s based on the Node-RED frontend, but uses its own engine to process the messages. 
-If you’re familiar with Node-Red, you won’t have any difficult to use it.
+It’s based on the Node-RED frontend, but uses its own engine to process the
+messages. If you’re familiar with Node-Red, it won't be difficult to use it.
 
 Why should I use it?
 ^^^^^^^^^^^^^^^^^^^^
 
-It allows one of the coolest things of IoT in an easy and intuitive way, which is to analyse data 
-for extracting information, then take actions.
+It allows one of the coolest things of IoT in an easy and intuitive way, which
+is to analyse data for extracting information, then take actions.
 
 What can it do, exactly?
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can do things such as:
 
-	- Create virtual viewers of a device (rename attributes, aggregate attributes, 
-	  change values, etc).
-
-	- Infer information based on switch rules.
-
-	- Infer information based on edge-detection rules.
-
-	- Infer information based on geo-fence rules.
-	
-	- Notify through email.
-	
-	- Notify through HTTP.
+  - Create virtual viewers of a device (rename attributes, aggregate
```

Comment:
> viewers

watchers

Reply:
I think the intention here was to create *views*, not *viewers*.